x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -189,10 +189,25 @@
                     null, MetadataUtils.DEFAULT_RESERVED_METADATA))
                 .put("apm_user", new RoleDescriptor("apm_user",
                     null, new RoleDescriptor.IndicesPrivileges[] {
+                        // Self managed APM Server
+                        // Can be removed in 8.0
                         RoleDescriptor.IndicesPrivileges.builder().indices("apm-*")
                             .privileges("read", "view_index_metadata").build(),
+                        
+                        // APM Server under fleet (data streams)
```

Comment:
Changes to the `apm_user` role LGTM - can we update the test for this role accordingly:

https://github.com/elastic/elasticsearch/blob/157152757545ebe2112d8750a397544f619f7166/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java#L933-L950

Reply:
I'm having a bit of trouble running the tests for Elasticsearch. Firstly, I don't understand the failures I'm seeing on CI.
Secondly, trying to follow [the docs ](https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc) to run the tests:
```
./gradlew :server:test --tests org.elasticsearch.xpack.core.security.authz.store
```

Just results in:
```
> Task :server:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':server:test'.
> No tests found for given includes: [**/*Tests.class](include rules) [**/*$*.class](exclude rules) [org.elasticsearch.xpack.*](--tests filter)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

Any clue?

Reply:
It seems like I can run non x-pack tests:
```
./gradlew :server:test --tests "org.elasticsearch.*"
```

Reply:
Ah I just figured it out:
```
./gradlew :x-pack:plugin:core:test --tests 'org.elasticsearch.xpack.core.security.authz.store.*'
```